cg/apps/housekeeper/hk.py

```diff
@@ -573,3 +575,50 @@ def store_fastq_path_in_housekeeper(
             bundle_name=sample_internal_id,
             tag_names=[SequencingFileTag.FASTQ, flow_cell_id, sample_internal_id],
         )
+
+    def get_archive_entries(
+        self, archival_task_id: int = None, retrieval_task_id: int = None
+    ) -> list[Archive]:
```

Comment:
Is it possible that this returns None?
```suggestion
    ) -> list[Archive] | None:
```

Reply:
I think it would return an empty list rather than None in that case, but I can double check!

Reply:
ahh ok, then feel free to ignore

Reply:
Empty list!